Feature/rename project

jay-79 opened the pull request:
Fix and merge PR#8 and PR#9

vincent314 committed:
chores: rules key are renamed from ECXXX to GCIXXX

vincent314 committed:
chores: rename all ecoCode instances to creedengo

vincent314 committed:
chore: set version to 2.0.0-SNAPSHOT

jay-79 committed:
feat: upgrade dependencies and prepare CHANGELOG for release 2.1.0

vdebellabre reviewed (changes requested):
Reste le répertoire src/main/resources/io/ecocode à renommer

jay-79 commented:
> Reste le répertoire src/main/resources/io/ecocode à renommer

Bien vu, je m'en occupe.

jay-79 commented:
> Reste le répertoire src/main/resources/io/ecocode à renommer

Non, c'est déjà fait. Il contient le fichier configuration.xml qui a été déplacé dans src/main/resources/org/greencodeinitiative/creedengo.

jay-79 committed:
feat: update default .net SDK to v9.0 on dotnetscan wrapper tool

jay-79 committed:
fix: align rules loader to embedded resources path from creedengo-rules-specification

jay-79 committed:
chore: markdown lint warning

jay-79 committed:
fix: unit test failed due to wrong mock path

jay-79 committed:
feat: update min java version and disable sonarlint supported to fix #10

Sonarlint could be disable for this plugin as bug could be directly reported by creedengo nuget plugin
